internal/store/postgres/org_billing_repository.go

```diff
@@ -403,11 +403,12 @@ func processStringDataType(filter rql.Filter, query *goqu.SelectDataset) *goqu.S
 		query = query.Where(goqu.L(fmt.Sprintf("coalesce(%s, '') = ''", filter.Name)))
 	case OPERATOR_NOT_EMPTY:
 		query = query.Where(goqu.L(fmt.Sprintf("coalesce(%s, '') != ''", filter.Name)))
-	case OPERATOR_IN, OPERATOR_NOT_IN:
-		// process the values of in and notin operators as comma separated list
-		query = query.Where(goqu.Ex{
-			filter.Name: goqu.Op{filter.Operator: strings.Split(filter.Value.(string), ",")},
-		})
+	case OPERATOR_IN:
```

Comment:
can we add these operator constants in rql since these are defined in RQL ?

Reply:
These are already added: https://github.com/raystack/salt/blob/main/rql/parser.go#L12

Reply:
No, I meant as constants so that they can be referred instead of each rql user implementing them.